libbeat/docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,268 @@
+{metricbeat}.
+endif::[]
+To learn about monitoring in general, see 
+{stack-ov}/xpack-monitoring.html[Monitoring the {stack}].
+
+//NOTE: The tagged regions are re-used in the Stack Overview.
+
+To collect and ship monitoring data:
+
+. <<configure-shipper,Configure the shipper you want to monitor>>
+
+. <<configure-metricbeat,Install and configure {metricbeat} to collect monitoring data>>
+
+[float]
+[[configure-shipper]]
+=== Configure the shipper you want to monitor
+
+. Enable the HTTP endpoint to allow external collection of monitoring data:
++
+--
+// tag::enable-http-endpoint[]
+Add the following setting in the {beatname_uc} configuration file
+(+{beatname_lc}.yml+):
+
+[source,yaml]
+----------------------------------
+http.enabled: true
+----------------------------------
+
+By default, metrics are exposed on port 5066. If you need to monitor multiple
+{beats} shippers running on the same server, set `http.port` to expose metrics
+for each shipper on a different port number:
+
+[source,yaml]
+----------------------------------
+http.port: 5067
+----------------------------------
+// end::enable-http-endpoint[]
+--
+
+. Disable the default collection of {beatname_uc} monitoring metrics. +
++
+--
+// tag::disable-beat-collection[]
+Add the following setting in the {beatname_uc} configuration file
+(+{beatname_lc}.yml+): 
+
+[source,yaml]
+----------------------------------
+monitoring.enabled: false
+----------------------------------
+// end::disable-beat-collection[]
+
+For more information, see 
+<<configuration-monitor,Monitoring configuration options>>.
+--
+
+ifndef::serverless[]
+. <<{beatname_lc}-starting,Start {beatname_uc}>>.
+endif::serverless[]
+
+[float]
+[[configure-metricbeat]]
+=== Install and configure {metricbeat} to collect monitoring data
+
+ifeval::["{beatname_lc}"!="metricbeat"]
+. {metricbeat-ref}/metricbeat-installation.html[Install {metricbeat}] on the
+same server as {beatname_uc}. If you already have {metricbeat} installed on the
+server, skip this step.
+endif::[]
+ifeval::["{beatname_lc}"=="metricbeat"]
+. The next step depends on how you want to run {metricbeat}:
+* If you're running as a service and want to run a separate monitoring instance,
+take the the steps required for your environment to run two instances of
+{metricbeat} as a service. The steps for doing this vary by platform and are
+beyond the scope of this documentation.
+* If you're running the binary directly in the foreground and want to run a
+separate monitoring instance,
+{metricbeat-ref}/metricbeat-installation.html[install {metricbeat}] to a
+different path. If necessary, set `path.config`, `path.data`, and `path.log`
+to point to the correct directories. See <<directory-layout>> for the default
+locations.
+* If you're using {metricbeat} to monitor itself, stop {metricbeat} and apply
+the following steps to your existing installation.
+endif::[]
+
+. Enable the beat-xpack module in {metricbeat}. +
```

Comment:
Should "beat-xpack" be enclosed in backticks?

Reply:
TBH, we're so inconsistent across the docs (too many different people updating them) that I'm not sure the best approach here. I do like how the extra formatting helps the name stand out, though, so let's go with backticks.